distribution/bwc/build.gradle

```diff
@@ -64,8 +64,12 @@ if (enabled) {
   File checkoutDir = file("${buildDir}/bwc/checkout-${bwcBranch}")
 
   task createClone(type: LoggedExec) {
-    onlyIf { checkoutDir.exists() == false }
-    commandLine = ['git', 'clone', rootDir, checkoutDir]
+    if (checkoutDir.exists() == false) {
+      commandLine = ['git', 'clone', rootDir, checkoutDir]
+    } else {
+      commandLine = ['git', 'fetch', 'origin']
```

Comment:
I'm not sure I follow - the previous logic checked if the directory existed and skipped cloning?